libcodechecker/analyze/log_parser.py

```diff
@@ -60,9 +60,34 @@ def get_compiler_includes(compiler, extra_opts=None):
                 do_append = True
 
     except OSError as oerr:
-        LOG.error("Cannot find include paths:" + oerr.strerror+"\n")
+        LOG.error("Cannot find include paths: " + oerr.strerror + "\n")
     return include_paths
 
+def get_compiler_target(compiler):
+    """
+    Returns the target triple of the given compiler as a string.
+    """
+    target_label = "Target:"
+    cmd = compiler + " " + "-v"
```

Comment:
No need to use two strings here, and this is a machine string and not a human string.

```py
    cmd = compiler + ' -v'
```